app/views/products/show.html.erb

```diff
@@ -0,0 +1,82 @@
+<div class="container">
+  <%= render 'layouts/navbar' %>
+
+  <% if !flash[:error].nil? %>
```

Comment:
Tends to be easier to read/understand `unless thing` rather than `if !thing`, especially when combined with `.nil?` or `.empty?`